dts-shared-services/openshift/templates/backup-postgres/backup-postgres-deploy.param

```diff
@@ -0,0 +1,41 @@
+#=========================================================
+# OpenShift template parameters for:
+# Component: .
+# Template File: ../openshift/templates/backup/backup-deploy.yaml
+#=========================================================
+NAME=backup-postgres
+SOURCE_IMAGE_NAME=backup-postgres
+NAMESPACE_NAME=a99fd4
+ROLE=backup-container
+SVC_PORT=5432
+APP_NAME=backup-container
+IMAGE_NAMESPACE=a99fd4-tools
+TAG_NAME=dev
+DATABASE_SERVICE_NAME=identity-kit-wallet-bc
+DATABASE_NAME=identity_kit_wallet
+TABLE_SCHEMA=public
+BACKUP_STRATEGY=rolling
+WEBHOOK_URL=
+ENVIRONMENT_FRIENDLY_NAME=Digital Trust Demo Apps (dev)
+ENVIRONMENT_NAME=a99fd4-dev
+DATABASE_DEPLOYMENT_NAME=identity-kit-wallet-bc
```

Comment:
See comment on the mongo template about speaking env names. All of the extra wallets being backed up are missing here by the way (see deployed instance).

Reply:
No, we should have them in the configuration otherwise we will likely be forgetting things. Plus the settings are in `backup.conf` already, so they should be added in all templates where necessary.